usr.sbin/nfsd/nfsd.c

```diff
@@ -999,7 +1008,10 @@ start_server(int master, struct nfsd_nfsd_args *nfsdargp)
 	struct stat st;
 
 	status = 0;
-	gethostname(hostname, sizeof (hostname));
+	if (vhost == NULL)
+		gethostname(hostname, sizeof (hostname));
+	else
+		strlcpy(hostname, vhost, sizeof (hostname));
```

Comment:
Just a minor, style(9) does not use space after sizeof.

Reply:
This isn't kernel, and I used the same spacing as the line that I replaced.

Reply:
I don't mind much about it.  Though style(9) applies to user space too, when possible.